infra/gp-kasten-instance/templates/01-pvc.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.kasten.backuplocation.enabled -}}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ .Values.kasten.backuplocation.claimname }}
```

Comment:
Achtung, in den Values heißt die Variable "claimName" mit großem N

Reply:
danke. gut gesehen!